### ISSUE TITLE
Create Profiling dev-docs section

### ISSUE THREAD

**Issue by DanielMurphy22** [mantidproject/mantid#29337]
Move the text in this commit to a new profiling section, to be inside Tools in the dev-docs.

See the 2 profiling sections already there: http://developer.mantidproject.org/ToolsOverview.html#profiling

Text to move: https://github.com/mantidproject/mantid/commit/0389b7b3bbb56ea02c98d0150fed31dfca59e3f3

**Comment by DanielMurphy22** [comment]
Text is:
```
Profiling
=========

Mantid supports profiling of its underlying code using the `cProfile <https://docs.python.org/3/library/profile.html>`_ module from python.

Launch Mantid Workbench from the command line using the script in the installation folder, adding the ``--profile`` modifier with a path to the output file.

E.g: ``path/to/install/bin/workbench --profile path/to/outputFile.prof``

This profile can then either be sent to the developers upon request or be analysed using the python module `snakeviz <https://pypi.org/project/snakeviz/>`_.
```

**Comment by DanielMurphy22** [comment]
Fixed in #30515
